src/RollbarSourceMapPlugin.js

```diff
@@ -164,9 +170,35 @@ class RollbarSourceMapPlugin {
       process.stdout.write('\n');
     }
     return Promise.all(
-      assets.map(asset => this.uploadSourceMap(compilation, asset))
+      assets.map(asset => this.uploadSourceMapWithRetry(compilation, asset))
     );
   }
+
+  async uploadSourceMapWithRetry(compilation, asset, depth = 0) {
+    try {
+      // eslint-disable-next-line no-console
+      console.info(`Uploading ${asset.sourceMap} to Rollbar`);
```

Comment:
Would you mind wrapping this in a `if (!this.silent)` check for consistency?

Reply:
Ah, yes! Updated.